args: Namespace(batch_size=400, cuda=True, data_path='./datasets/amazon-book_noisy/', dataset='amazon-book_noisy', gpu='1', log_name='log', mean_type='x0', noise_max=0.0005, noise_min=0.0001, noise_scale=0.005, noise_schedule='linear-var', num_workers=0, sampling_noise=False, sampling_steps=0, steps=10, topN='[10, 20, 50, 100]', tst_w_val=False)
Starting time:  2023-06-12 14:29:17
user num: 108822
item num: 178181
data ready.
Traceback (most recent call last):
  File "./DiffRec/inference.py", line 207, in <module>
    model = torch.load(model_path + model_name).to(device)
  File "/home/lcur2470/.conda/envs/rs/lib/python3.8/site-packages/torch/serialization.py", line 777, in load
    with _open_zipfile_reader(opened_file) as opened_zipfile:
  File "/home/lcur2470/.conda/envs/rs/lib/python3.8/site-packages/torch/serialization.py", line 282, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory
